infrastructure/main.tf

```diff
@@ -152,10 +162,17 @@ module "case-service" {
     NOTIFY_API_KEY                                      = "${data.azurerm_key_vault_secret.notify_api_key.value}"
     FPL_SYSTEM_UPDATE_USERNAME                          = "${data.azurerm_key_vault_secret.system_update_user_username.value}"
     FPL_SYSTEM_UPDATE_PASSWORD                          = "${data.azurerm_key_vault_secret.system_update_user_password.value}"
+    ROBOTICS_NOTIFICATION_SENDER                        = "${data.azurerm_key_vault_secret.robotics-notification-sender}"
+    ROBOTICS_NOTIFICATION_RECIPIENT                     = "${data.azurerm_key_vault_secret.robotics-notification-recipient}"
     SPRING_SECURITY_ENABLED                             = "${var.security_enabled}"
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI = "${var.idam_token_issuer_uri}"
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI = "${var.idam_token_jwk_set_uri}"
+    SPRING_MAIL_HOST                                    = "${var.mail_host}"
+    SPRING_MAIL_PORT                                    = "${var.mail_port}"
+    SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE    = "${var.mail_properties_smtp_starttls_enable}"
+    SPRING_MAIL_PROPERTIES_MAIL_SMTP_SSL_TRUST          = "${var.mail_host}"
     GATEWAY_URL                                         = "${var.gateway_url}"
+    FEATURE_TOGGLE_ROBOTICS_CASE_NUMBER_NOTIFICATION_ENABLED    = "${var.feature_toggle_robotics_case_number_notification_enabled}"
```

Comment:
In Terraform world it is a good practice to align all equal signs.

Reply:
thanks @geek-soft. Now updated [here](https://github.com/hmcts/fpl-ccd-configuration/blob/7cb443a0301fb767118468141186119d23cf96ee/infrastructure/main.tf)